zuul.d/projects.yaml

```diff
@@ -6,3 +6,4 @@
         - dataplane-operator-docs-preview
     templates:
       - podified-multinode-edpm-baremetal-pipeline
+      - data-plane-adoption-pipeline
```

Comment:
as discussed in yesterday's adoption call:

I'll add one of the jobs here explicitly instead of adding the data-plane-adoption-pipeline template. That way we will only run one job instead of both (the job with ceph, non tls will run here) and we can restrict when this job will run using files: